test/NuGet.Core.Tests/NuGet.CommandLine.XPlat.Test/BasicLoggingTests.cs

```diff
@@ -69,6 +69,29 @@ public void BasicLogging_RestoreHelp_ExitCode()
         }
 
         [Fact]
+        public void BasicLogging_RestoreConfigFile_ExitCode()
+        {
+            // Arrange
+            var log = new TestLogger();
```

Comment:
I'm not sure how to exactly do this.  There isn't an existing tests that actually DOES a restore.  All the tests in this project either:
1. are negative tests
2. Check that "restore --help" returns 0 exit code
3. empty args returns 0 exit code.